ocaml/xapi/message_forwarding.ml

```diff
@@ -6554,6 +6554,40 @@ functor
 
     module Certificate = struct end
 
+    module Host_driver = struct
+      (** [create] creates a database object; this can be done on the
+          coordinator *)
+      let create ~__context ~host ~name_label ~versions ~active_version
+          ~selected_version =
+        info "%s" __FUNCTION__ ;
+        Xapi_host_driver.create ~__context ~host ~name_label ~versions
```

Comment:
Is it possible that a pool member receives this call? Do you need to have some kind of helper to always forward this call to the coordinator?

Reply:
I believe an API call is only accepted on the coordinator - but we can ask @robhoes if the forwarding mechanism is required.

Reply:
The `create` and `destroy` functions are not actually part of the API as defined in the datamodel, as `~gen_constructor_destructor:false` and the are not explicitly defined. And we don't need them, as xapi itself is meant to create and destroy these objects. So these functions (in message_forwarding.ml) will never be executed.